cmd/scripts/k8s/setup-host.sh

```diff
@@ -22,26 +22,24 @@ env \
     CEDANA_OTEL_ENABLED="$CEDANA_OTEL_ENABLED" \
     CEDANA_OTEL_PORT="$CEDANA_OTEL_PORT" \
     CEDANA_LOG_LEVEL="$CEDANA_LOG_LEVEL" \
+    SKIPSETUP="$CEDANA_SKIPSETUP" \
     chroot /host /bin/bash <<'EOT'
 
 if [[ $SKIPSETUP -eq 1 ]]; then
     cd /
-    CEDANA_LOG_LEVEL="$CEDANA_LOG_LEVEL" CEDANA_URL="$CEDANA_URL" CEDANA_AUTH_TOKEN="$CEDANA_AUTH_TOKEN" ./build-start-daemon.sh --systemctl --no-build --otel --k8s
```

Comment:
why do we ignore these variables if we're skipping setup?

Reply:
we don't use them for normal startup

Reply:
they should already be present

Reply:
SKIPSETUP was never set to 1 for default installs